src/proto/FIO.proto

```diff
@@ -150,11 +150,25 @@ message SigningInput {
     Action action = 5;
 }
 
+enum ErrorCode {
```

Comment:
can FIO reuse EOS `ErrorCode`?

Reply:
now there is one common def, each chain references it